src/riscv/linux/hwcap.c

```diff
@@ -0,0 +1,37 @@
+#include <riscv/linux/api.h>
+
+#if CPUINFO_MOCK
+    #include <cpuinfo-mock.h>
+#endif
+
+#if CPUINFO_ARCH_RISCV32 || CPUINFO_ARCH_RISCV64
+    #include <sys/auxv.h>
+#else
+    #define AT_HWCAP 16
+    #define AT_HWCAP2 26
```

Comment:
Do we need HWCAP2? I don't think it does anything on RISC-V. I guess I'm also a bit confused as to why the ARCH here determines how AT_HWCAP is defined - these variable do not seem to be used at all?